<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMCellStyleWidget</class>
 <widget class="QWidget" name="SMCellStyleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>478</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="smColorGroup">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="fillColorIcon">
          <property name="maximumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="buddy">
           <cstring>fillColor</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMColorCombo" name="fillColor"/>
        </item>
        <item>
         <widget class="QLabel" name="fillShadeLabel">
          <property name="text">
           <string>Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SMShadeButton" name="fillShade_">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>72</width>
       <height>388</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SMColorCombo</class>
   <extends>QComboBox</extends>
   <header>ui/smcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>SMShadeButton</class>
   <extends>QPushButton</extends>
   <header>ui/smshadebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
